src/Model/ModelUtilities/GwfSfrStatic.f90

```diff
@@ -0,0 +1,70 @@
+  !> @brief Kinematic routing equation residual
```

Comment:
What does "static" mean in this filename?  Should kinematic routines live in file named static?  Should it be "common"?

Reply:
Renamed source file to `GwfSfrCommon.f90`